assembl/static2/js/app/components/debate/survey/question.jsx

```diff
@@ -105,14 +91,15 @@ class Question extends React.Component {
     }
   }
   render() {
-    const { index, title } = this.props;
+    const { index, title, screenWidth, screenHeight } = this.props;
+    const height = screenHeight - document.getElementById('timeline').clientHeight;
```

Comment:
I would like to get rid of `document`/`window` stuff in components. `timeline` could be set in context or in redux store by the root app component (it's probably something we should discuss)

Reply:
I won't do that in this PR unless you want me to

Reply:
in fact here, we should probably use a `ref`. I don't have a clear idea of where things are in the DOM but I guess it's really not easy to pass timeline's ref to `Question`